pkg/controllers/cloud/networkpolicy.go

```diff
@@ -1159,13 +1160,15 @@ func (a *appliedToSecurityGroup) notifyNetworkPolicyChange(r *NetworkPolicyRecon
 }
 
 // removeStaleMembers removes sg members that their corresponding CRs no longer exist and cleans up relevant internal resources.
+// we do not make cloud api calls to update members because we do not know if vm is still present in cloud or not.
```

Comment:
Dont use we.. Rephrase.. No cloud api calls will be made to update members because VM may be terminated in cloud